packages/paste-website/src/pages/components/toggle/index.mdx

```diff
@@ -0,0 +1,144 @@
+  githubUrl="https://github.com/twilio-labs/paste/tree/main/packages/paste-core/components/toggle"
+  storybookUrl="/?path=/story/components-toggle--default"
+  data={props.data}
+  description={props.data.mdx.frontmatter.description}
+/>
+
+---
+
+<contentwrapper>
+
+<PageAside data={props.data.mdx} />
+
+<content>
+
+## Guidelines
+
+### About Toggle
+
+A Toggle is an interactive binary switch. It should be used in forms when an "on/off" or "yes/no" input is needed. Use label text to describe the purpose of the Toggle. The label text shouldn't change when the state of the Toggle changes. Help Text can also be used to provide additional guidance.
+
+### Accessibility
+
+Toggle has `role="switch"`. The `aria-labelledby` attribute is automatically set on the component, which ensures that assisstive technology devices can read the Toggle's label. When the Toggle is on, `aria-selected` is set to `"true"` (it defaults to `"false"`).
+
+## Examples
+
+### Default
+
+<LivePreview scope={{Toggle, useUID}}>{basicToggle}</LivePreview>
+
+### Disabled
+
+<LivePreview scope={{Toggle, useUID}}>{disabledToggle}</LivePreview>
+
+## States
+
+### On
+
+The default Toggle state is "off"
+
+<LivePreview scope={{Toggle, useUID}}>{onToggle}</LivePreview>
+
+## Usage Guide
+
+### API
+
+#### Installation
+
+```bash
+yarn add @twilio-paste/toggle - or - yarn add @twilio-paste/core
+```
+
+#### Usage
+
+```jsx
+import {Toggle} from '@twilio-paste/core/toggle';
+
+export const Basic = () => {
+  return <Toggle>Label</Toggle>;
+};
+```
+
+#### Props
+
+| Prop        | Type                           | Description                                                                               | Default    |
+| ----------- | ------------------------------ | ----------------------------------------------------------------------------------------- | ---------- |
+| `children`  | `NonNullable<React.ReactNode>` | Pass the label text as children                                                           |            |
+| `element?`  | `string`                       | Overrides the default element name to apply unique styles with the Customization Provider | `'TOGGLE'` |
+| `disabled?` | `boolean`                      |                                                                                           | `false`    |
+| `helpText?` | `string` or `ReactNode`        |                                                                                           | `null`     |
+| `id?`       | `string`                       |                                                                                           | `null`     |
+| `on?`       | `boolean`                      | Controls the state of the Toggle                                                          | `off`      |
+| `required?` | `boolean`                      |                                                                                           | `false`    |
+
+### Figma
```

Comment:
Suggestion: since it's not in figma yet, you can either remove the section or add a callout saying it is coming to figma soon?

Reply:
yes i removed it! making some other additions to the docs page too